GUIDE.md

```diff
@@ -0,0 +1,203 @@
+one translation has still not arrived from ePoetry. If all have arrived, then the flow is as usual (you can make a new request).
+
+The `Update` translation request form will present you the same fields as when making the initial translation request. However, when it is sent successfully to ePoetry, it replaces the original translation request which is marked as `Finished` in its Request Status. The two requests
+are linked together and this link is visible in the Log Messages of both requests, as well as on the new translation request page.
+
+*NOTE: when using the corporate workflow, a new version of the content means that you have created a new Validated revision.*
+
+#### Translation review
+
+When a given language translation is in Review, you can click the Review button for that language and check the translation values. You can then do the following:
+
+* Accept the translation -> saves the request with this translation marked as Accepted
+* Synchronise the translation -> synchronises the translation with the Node
+
+Note: if the request has Auto-accept or Auto-sync turned on, the above steps would happen automatically as the translation arrives from ePoetry.
+
+#### Failed requests
+
+If the translation request fails and there is an error shown, you need to contact the site administrator. The request is currently marked as Failed and the Log Messages on the request page should
+indicate the error. Once the issue was addressed, the failed request must be marked as Failed & Finished. You can do so by going to the translations
+dashboard and using the `Mark as finished` operation on the respective translation request. This will unblock the flow and a request can be tried again.
+
+### Poetry legacy
+
+For sites that have used already Poetry, there is a Poetry legacy table for administrators who can check the last Poetry reference that was sent for each node.
+
+Dashboard: `/admin/content/legacy-poetry-references`
+
+Moreover, the first time an ePoetry request is being sent, the Message value will include automatically the last Poetry reference ID of that Node so that ePoetry can find it back from the old system.
+
+### ePoetry translations dashboard
+
+Apart from the individual Node translations dashboard, you also have a dashboard where you can have an overview over all the ePoetry translation requests:
+
+`/admin/content/epoetry-translation-requests`
+
+## Synchronisation
+
+Synchronisation means that the translation values will be saved onto the Node itself (creating or updating the translation).
+
+Each translation request (both local or remote) is started from a specific Node version. This means that when the synchronisation happens, the translation values
+get saved onto that specific revision. This means that if you have ongoing translations and you make a new draft, those translations will be synced onto the previous revision, the new draft
```

Comment:
version instead of revision, no? in case we published the validated in the meantime.

Reply:
OK I see the workflow specific part below.